.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,36 @@
+name: CI/CD Pipeline
```

Comment:
Can we call this `build`. As far as I can see this is a little bit like a convention and will make a nice badge 😊

Reply:
And maybe also call the file build.yml